documentation/overview.md

```diff
@@ -103,101 +103,101 @@ Note also that all Doc Actions are themselves valid User Actions, i.e. User Acti
     Contains modules responsible for maintaining client-side data.
-    * `app/client/models/`**`TableData.ts`**, `app/client/models/`**`DocData.ts`**
-        Enhancements of same-named classes in `app/common` (see above) which add some client-side functionality like helpers to send User Actions.
-    * `app/client/models/`**`DocModel.ts`**
-        Maintains *observable* data models, for all metadata and user data tables in a document. For metadata tables, the individual records are enhanced to be specific to each type of metadata, using classes in `app/client/models/entities`. For example, `docModel.columns` is a `MetaTableModel` containing records of type `ColumnRec` (from `app/client/models/entities/ColumnRec.ts`) which are derived from `MetaRowModel`.
-    * `app/client/models/`**`TableModel.js`**
+    * [`app/client/models/`**`TableData.ts`**](../app/client/models/TableData.ts), [`app/client/models/`**`DocData.ts`**](../app/client/models/DocData.ts)
+        Enhancements of same-named classes in [`app/common`](../app/common/) (see [above](#common)) which add some client-side functionality like helpers to send User Actions.
+    * [`app/client/models/`**`DocModel.ts`**](../app/client/models/DocModel.ts)
+        Maintains *observable* data models, for all metadata and user data tables in a document. For metadata tables, the individual records are enhanced to be specific to each type of metadata, using classes in [`app/client/models/entities`](../app/client/models/entities/). For example, `docModel.columns` is a `MetaTableModel` containing records of type `ColumnRec` (from [`app/client/models/entities/ColumnRec.ts`](../app/client/models/entities/ColumnRec.ts)) which are derived from `MetaRowModel`.
+    * [`app/client/models/`**`TableModel.js`**](../app/client/models/TableModel.js)
         Base class for `MetaTableModel` and `DataTableModel`. It wraps `TableData` to make the data observable, i.e. to make it possible to subscribe to changes in it. This is the basis for how we build most UI.
-    * `app/client/models/`**`MetaTableModel.js`**
+    * [`app/client/models/`**`MetaTableModel.js`**](../app/client/models/MetaTableModel.js)
         Maintains data for a metadata table, making it available as observable arrays of `MetaRowModel`s. The difference between metadata tables and user tables is that the Grist app knows what’s in metadata, and relies on it for its functionality. We also assume that metadata tables are small enough that we can instantiate [observables](https://github.com/gristlabs/grainjs/blob/master/docs/basics.md#observables) for all fields of all rows.
-    * `app/client/models/`**`DataTableModel.js`**
+    * [`app/client/models/`**`DataTableModel.js`**](../app/client/models/DataTableModel.js)
         Maintains data for a user table, making it available as `LazyArrayModel`s (defined in the same file), which are used as the basis for `koDomScrolly` (see `app/client/lib/koDomScrolly.js` below).
```

Comment:
You might add a link here too:

```suggestion
        Maintains data for a user table, making it available as `LazyArrayModel`s (defined in the same file), which are used as the basis for `koDomScrolly` (see [`app/client/lib/koDomScrolly.js`](/app/client/lib/koDomScrolly.js) below).
```

Reply:
I actually specifically didn't add it to this one since this refers to below, which has both a link and a description. As in, linking this one may make a reader miss the description, which goes against the intent of the text